packages/back-end/src/models/AITokenUsageModel.ts

```diff
@@ -0,0 +1,66 @@
+import { omit } from "lodash";
+import mongoose from "mongoose";
+import { AITokenUsageInterface } from "../../types/ai";
+import { OrganizationInterface } from "../../types/organization";
+
+type AITokenUsageDocument = mongoose.Document & AITokenUsageInterface;
+
+const RESET_INTERVAL = 24 * 60 * 60 * 1000; // 24 hours
+
+const aiTokenUsageSchema = new mongoose.Schema({
+  id: String,
+  organization: String,
```

Comment:
Should we mark this as unique just to be sure?

Reply:
Yes, updated